File: app/src/main/java/com/digicoffer/lauditor/Groups/Models/ActionModel.java
package com.digicoffer.lauditor.Groups.Models;

public class ActionModel {
    private String name;
    private boolean isEnabled;

    public ActionModel(String name) {
        this.name = name;
        this.isEnabled = true; // default to enabled
    }

    public ActionModel(String name, boolean isEnabled) {
        this.name = name;
        this.isEnabled = isEnabled;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public boolean isEnabled() {
        return isEnabled;
    }

    public void setEnabled(boolean enabled) {
        isEnabled = enabled;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Groups/Models/ActionModel.kt
package com.digicoffer.lauditor.Groups.Models

class ActionModel {
    var name: String = ""
    var isEnabled: Boolean = true

    constructor(name: String) {
        this.name = name
        this.isEnabled = true
    }

    constructor(name: String, isEnabled: Boolean) {
        this.name = name
        this.isEnabled = isEnabled
    }
}
